src/proc.py

```diff
@@ -29,9 +29,24 @@
 from subprocess import * # flake8: noqa
 
 
+def FixPythonAndWindows(cmd):
+  if cmd[0].endswith('.py'):
+    return [sys.executable] + cmd
+
+  exe = cmd[0]
+  def IsExe(f):
```

Comment:
Actually this isn't going to work unless we also essentially implement `which` by walking through PATH, or using `shell=True` which would mean we wouldn't need to use the file extensions anyway. Neither of those seems very appealing to me.